File: app/src/main/java/com/afoxxvi/alopex/notify/NotifyStorage.java
package com.afoxxvi.alopex.notify;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

import androidx.annotation.Nullable;

public class NotifyStorage extends SQLiteOpenHelper {
    private static final int VERSION = 1;

    public NotifyStorage(@Nullable Context context, @Nullable String name) {
        super(context, name, null, VERSION);
    }

    private static final String CREATE_PACKAGE = "" +
            "create table Package(" +
            "id integer primary key autoincrement," +
            "package text" +
            ");";

    private static final String CREATE_NOTIFY = "" +
            "create table Notify(" +
            "package integer," +
            "title text," +
            "content text," +
            "time datetime" +
            ");";

    @Override
    public void onCreate(SQLiteDatabase db) {
        db.execSQL(CREATE_PACKAGE);
        db.execSQL(CREATE_NOTIFY);
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {

    }
}

File: app/src/main/java/com/afoxxvi/alopex/component/notify/NotifyStorage.kt
package com.afoxxvi.alopex.component.notify

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

class NotifyStorage(context: Context?, name: String?) : SQLiteOpenHelper(context, name, null, VERSION) {
    override fun onCreate(db: SQLiteDatabase) {
        db.execSQL(CREATE_PACKAGE)
        db.execSQL(CREATE_NOTIFY)
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {}

    companion object {
        private const val VERSION = 1
        private const val CREATE_PACKAGE = "" +
                "create table Package(" +
                "id integer primary key autoincrement," +
                "package text" +
                ");"
        private const val CREATE_NOTIFY = "" +
                "create table Notify(" +
                "package integer," +
                "title text," +
                "content text," +
                "time datetime" +
                ");"
    }
}